modules/sdk-core/src/bitgo/tss/eddsa/eddsa.ts

```diff
@@ -217,6 +201,56 @@ export async function offerUserToBitgoRShare(
   );
 }
 
+export const delimiter = ':';
+
+export function convertRShare(rShare: RShare): SignatureShareRecord {
+  return {
+    from: getParticipantFromIndex(rShare.j),
+    to: getParticipantFromIndex(rShare.i),
+    share: `${rShare.r}${delimiter}${rShare.R}${delimiter}${rShare.u}${delimiter}${rShare.v ? rShare.v : ''}`,
+  };
+}
```

Comment:
why do we want to introduce a delimiter? this becomes a break / backward incompatible change

Reply:
This is a refactor I thought will help reduces duplicate code. It is breaking change however since we are breaking change as part vss, we should include the refactor as well.

Reply:
reverted this changes